src/amber.cr

```diff
@@ -34,16 +21,37 @@ module Amber
       instance
     end
 
+    def self.key_generator
+      instance.key_generator
+    end
+
```

Comment:
Oh ok. I might be wrong on this but it seems like properties, readers, writers should always be before methods.